core/include/traccc/seeding/detail/seeding_config.hpp

```diff
@@ -114,7 +114,8 @@ struct seedfinder_config {
 
     TRACCC_HOST_DEVICE
     unsigned int get_max_neighbor_bins() const {
-        return std::pow(neighbor_scope[0] + neighbor_scope[1] + 1, 2);
+        return static_cast<unsigned int>(
```

Comment:
If `neighbor_scope` are integers, this is a bad solution. This should be done using integer operations only, e.g.:

```suggestion
        unsigned int t = neighbor_scope[0] + neighbor_scope[1] + 1;
        return t * t;
```

Reply:
:+1: Absolutely what should be done here.